multibase_test.go

```diff
@@ -81,8 +83,9 @@ func TestDecode(t *testing.T) {
 func TestRoundTrip(t *testing.T) {
 	buf := make([]byte, 17)
 	rand.Read(buf)
+	fmt.Println(buf)
```

Comment:
Best to avoid printf in tests. If you want to log this, please use `t.Log`.

Reply:
This was by a mistake. Removed now.